WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardBloganuaryCardCell.swift

```diff
@@ -163,7 +163,7 @@ private struct BloganuaryNudgeCardView: View {
         static let description = NSLocalizedString(
             "bloganuary.dashboard.card.description",
             value: """
-            For the month of January, blogging prompts will come from Bloganuary - \
+            For the month of January, blogging prompts will come from Bloganuary — \
             our community challenge to build a blogging habit for the new year.
             """,
```

Comment:
Given the red background, it appears GitHub thinks this is a syntax error. The file compiles fine though, but I can change this to a single line if we'd like.

Reply:
Decided to just update it: https://github.com/wordpress-mobile/WordPress-iOS/pull/22213/commits/6155351482548b5ebd768a36b82eb281cdc81b98.